app/api/currentuser/send_group_leave_request.go

```diff
@@ -0,0 +1,48 @@
+package currentuser
+
+import (
+	"net/http"
+
+	"github.com/France-ioi/AlgoreaBackend/app/service"
+)
+
+// swagger:operation POST /current-user/group-leave-requests/{group_id} groups users groupLeaveRequestCreate
+// ---
+// summary: Create a request to leave a group
+// description: >
+//   Lets a user create a request to leave a group.
+//
+//     On success the service creates a new row in `group_pending_requests` with `group_id` = `{group_id}`
+//     `type` = 'leave_request' and `member_id` = user's `group_id` + a new row in `group_membership_changes`
+//     with `action` = `leave_request_created` and `at` equal to current UTC time.
+//
+//     * If there is already a row in `groups_groups` or a row in `group_pending_request` with
```

Comment:
"If there is already a row in `groups_groups`" -> not a row? Having a row is normal for leave requests. Maybe your 'or' should be a "and"?

Reply:
fixed